.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,7 +32,9 @@ What did you see instead?
 ## Reproduce
 
 Create a self-contained project using the template of your choice, apply the
-minimum required code to result in the issue you're observing.
+minimum required code to result in the issue you're observing. You may start
+from the apps in the [docs](../docs/) folder, and modify to demonstrate the
```

Comment:
Do we really need this?

Reply:
Neutral opinion actually.. Initially I was thinking to recommend to start from the examples and tweak it to repro.. maybe its not needed

Reply:
reverted that part. will comeback if i get better wording.